addons/l10n_fr_fec/wizard/fec.py

```diff
@@ -66,8 +66,9 @@ def do_query_unaffected_earnings(self):
             '''
         company = self.env.user.company_id
         formatted_date_from = self.date_from.replace('-', '')
+        formatted_date_year = formatted_date_from[0:4]
```

Comment:
It is better to do smth like this, that way if for some reason we decide to change the format of the date it won't crash

```
from datetime import datetime
from odoo.tools import DEFAULT_SERVER_DATE_FORMAT
date_from = datetime.strptime(self.date_from, DEFAULT_SERVER_DATE_FORMAT)
formatted_date_year = date_from.year
```